fastlane/fastlane/Fastfile

```diff
@@ -178,7 +186,7 @@ private_lane :print_release_changelog do |options|
   tool_name = options[:tool]
   old_version = options[:old_version]
 
-  cmd = "git log --pretty='* %B' #{tool_name}/#{old_version}...HEAD --no-merges .."
+  cmd = "git log --pretty='* %s' #{tool_name}/#{old_version}...HEAD --no-merges .."
```

Comment:
This change reduces the output to just the subject (first line) of the git commit.

'%B': raw body (unwrapped subject and body)
'%s': subject